models/varnet/varnet.py

```diff
@@ -254,14 +269,13 @@ def validation_step(self, batch, batch_idx):
         }
 
     def test_step(self, batch, batch_idx):
-        input, _, mean, std, fname, slice = batch
-        output = self.forward(input)
-        mean = mean.unsqueeze(1).unsqueeze(2)
-        std = std.unsqueeze(1).unsqueeze(2)
+        masked_kspace, mask, target, fname, slice, _ = batch
+        output = self.forward(masked_kspace, mask)
+        _, output = T.center_crop_to_smallest(target, output)
```

Comment:
I see now why you make the target args.resolution size. I think it's still better to make the target = None and then here use T.center_crop(args.resolution, args.resolution) to make it clear in the test function that we're not using a target.

Reply:
Yes I agree. Will make the edits accordingly.

Reply:
T.center_crop was added as suggested.